src/v0/destinations/am/deleteUsers.js

```diff
@@ -1,17 +1,19 @@
+const _ = require("lodash");
 const btoa = require("btoa");
-const { httpSend } = require("../../../adapters/network");
-const { getDynamicErrorType } = require("../../../adapters/utils/networkUtils");
+const { httpPOST } = require("../../../adapters/network");
+const tags = require("../../util/tags");
 const {
-  processAxiosResponse
+  processAxiosResponse,
+  getDynamicErrorType
 } = require("../../../adapters/utils/networkUtils");
 const { isHttpStatusSuccess } = require("../../util");
 const {
+  ConfigurationError,
   NetworkError,
-  InstrumentationError,
-  ConfigurationError
+  InstrumentationError
 } = require("../../util/errorTypes");
 const { executeCommonValidations } = require("../../util/regulation-api");
-const tags = require("../../util/tags");
```

Comment:
This import is not supposed to be deleted.

Reply:
Removed